### DISCUSSION TITLE
I can't subscribe to Copilot

### DISCUSSION THREAD

**Discussion by NickHeiner** [community/community#42423] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/829827/209212834-fbcd1b76-f58f-4c7c-ac7b-b8a6e9b7d392.png)

This is what the settings page looks like.

I'd like to upgrade to a paid account, but I can't figure out how.

**Comment by iamcjwilliams** [comment]
@NickHeiner Visit [here](https://docs.github.com/en/billing/managing-billing-for-github-copilot/managing-your-github-copilot-for-individuals-subscription) to manage your subscription
